data/load.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# http://www.manniwood.com/postgresql_and_bash_stuff/
+# Set these environmental variables to override them,
+# but they have safe defaults.
+export PGHOST=${PGHOST-localhost}
```

Comment:
shouldn't that be something like:

```
export PGHOST=${PGHOST:-localhost}
```

or, alternatively...

```
export PGHOST=${PGHOST:=localhost}
```

like aren't you missing the ':'?  does this work?